src/ontology/foodon-edit.owl

```diff
@@ -65857,17 +65872,22 @@ SubClassOf(obo:FOODON_03530217 obo:FOODON_03530008)
 
 # Class: obo:FOODON_03530218 (food packing medium integrity)
 
-AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530218 obo:IAO_0000428)
-AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530218 "It is important to distinguish whether the food in question has been drained or not drained when evaluating the nutrient content.
-Related term to PRESENCE OF PACKING MEDIUM is K.  PACKING MEDIUM."@en)
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03530218 "A can of peaches in syrup, in which the syrup is uncompromised."@en)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530218 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530218 "A food object quality which describes the integrity of the medium in which the food is packed for preservation and handling or the medium surrounding foods."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03530218 "Existing definition was a legacy definition, not ontological and has been moved to comments. Updated examples of usage to remove plastic usage as describes packaging not packing medium."@en)
```

Comment:
When mentioning `existing` you are now referencing the current definition, not the retired one, as the context of it's application changed when the definition did. Recommend using "Previous" like you did in the heading of your comment.

```suggestion
AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03530218 "Previous definition was a legacy definition, not ontological and has been moved to comments. Updated examples of usage to remove plastic usage as describes packaging not packing medium."@en)
```

Reply:
Resolved in later commits.